src/logic/wallets/transactionDataCheck.ts

```diff
@@ -1,18 +1,19 @@
+import { loadFromStorage, saveToStorage } from 'src/utils/storage'
+
+const LEDGER_TRANSACTION_DATA = 'TRANSACTION_DATA'
```

Comment:
can we make the key/name to be more concrete? what's transaction data? maybe `USER_ENABLED_LEDGER_TX_DATA`